Assets/Scripts/Levitatable.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using Cinemachine;
+using UnityEngine;
+
+[RequireComponent (typeof (Rigidbody))]
+public class Levitatable : MonoBehaviour, ICastable {
+    public string SpellId = "159AB";
+    public CinemachineVirtualCamera cam;
```

Comment:
Will this be used for anything? I saw it set to a move camera prefab. This is slightly concerning because if we make the move camera persist between scenes this reference would break.